packages/mobile/src/invite/saga.ts

```diff
@@ -216,6 +208,14 @@ export function* sendInvite(
     // Store the Temp Address locally so we know which transactions were invites
     yield put(storeInviteeData(inviteDetails))
 
+    if (features.ESCROW_WITHOUT_CODE) {
+      if (amount) {
+        yield call(initiateEscrowTransfer, e164Number, amount)
```

Comment:
I'm not sure I understand why changing the order of this fixes the issue. I see we're now calling `initiateEscrowTransfer` **after** calling `storeInviteeData`, is that the important part? If so, why does that change things and if not I don't understand the fix :P